bin/check-disk-usage.rb

```diff
@@ -63,6 +63,10 @@ class CheckDisk < Sensu::Plugin::Check::CLI
          description: 'Ignore option(s)',
          proc: proc { |a| a.split('.') }
 
+  option :ignorereadonly,
+         long: '--ignore-readonly'
+         description: 'Ignore read-only filesystems'
```

Comment:
Probably best to explicitly call out that its a boolean option:
```suggestion
         description: 'Ignore read-only filesystems',
         boolean:     true
```